functions/levels.py

```diff
@@ -20,9 +20,9 @@ async def level(self, ctx, *args):
             year = args[0]
             day = args[1]
         else:
-            today = datetime.now(self._timezone)
```

Comment:
today is needed to get current year in the next line tho

Reply:
oops lol